### ISSUE TITLE
It is not clear if anonymous volume is created for each `docker run --rm` or not?

### ISSUE THREAD

**Issue by EugenKon** [docker/docs#18317]
### Is this a docs issue?

- [X] My issue is about the documentation content or website

### Type of issue

I can't find what I'm looking for

### Description

>When you mount a volume, it may be named or anonymous. Anonymous volumes are not given an explicit name when they are first mounted into a container, so Docker gives them a random name that is guaranteed to be unique within a given Docker host. Besides the name, named and anonymous volumes behave in the same ways.

It is not clear for me if I `docker run -d --rm --name "test" -v /dbdata IMG; docker run -d --rm --name "test" -v /dbdata IMG` will `/dbdata` be given random name for each run or only for first mount and this random name will be reused for second run?

### Location

https://docs.docker.com/storage/#more-details-about-mount-types

### Suggestion

Describe the behaviour for mount command

**Comment by Saifullah00** [comment]
#18317
When we use the Docker `-v` or `--volume` option to mount a volume into a container, the specified volume name will be used consistently for subsequent runs of the same container as long as the volume name is the same.

In your provided example:
```shell
docker run -d --rm --name "test" -v /dbdata IMG
docker run -d --rm --name "test" -v /dbdata IMG
```

The volume name is `/dbdata`, and it will be consistently used for both runs of the "test" container. 
Docker will not assign a random name for each run as long as you specify the same volume name.

In other words, if you run the same `docker run` command multiple times with the same volume name, it will use the same volume for each run, and the data in that volume will be persistent across those runs.

However, if you change the volume name between runs, Docker will create a new volume with the new name for each run. For example:
```shell
docker run -d --rm --name "test" -v /dbdata1 IMG
docker run -d --rm --name "test" -v /dbdata2 IMG
```
In this case, two different volumes, `/dbdata1` and `/dbdata2`, will be created and used for the two runs.

**Comment by EugenKon** [comment]
@Saifullah00 Please notice, I did not give names to my volumes. [Please look here](https://docs.docker.com/storage/volumes/#choose-the--v-or---mount-flag):

>In the case of named volumes, the first field is the name of the volume, and is unique on a given host machine. For anonymous volumes, the first field is omitted.

As stated by documentation `Docker gives them a random name`.

>However, if you change the volume name between runs

Sorry, but your example is incorrect. 

```
docker run -d --rm --name "test" -v /dbdata1 IMG
docker run -d --rm --name "test" -v /dbdata2 IMG
```

You did not provide names for volumes, you just change path where anonymous volume will be mounted in the container.

**Comment by Saifullah00** [comment]
Sorry for the inconvenience.
